src/components/Card/Card.stories.tsx

```diff
@@ -3,11 +3,14 @@ import { Box } from '@chakra-ui/core';
 import { storiesOf } from '@storybook/react';
 import { boolean, select } from '@storybook/addon-knobs';
 
-import Card, { CardElevations, CardSizes } from './Card';
+import Card from './Card';
 import StoryHeading from '../StoryHeading/StoryHeading';
 import Text from '../Text/Text';
 
-const Container = props => <Box m='xl' {...props} />;
+const Container = (props: any) => <Box m='xl' {...props} />;
+
+const elevations = ['none', 's', 'm'] as const;
```

Comment:
Out of interest: what `as const` means?

Reply:
As explained in the PR description: https://stackoverflow.com/questions/37978528/typescript-type-string-is-not-assignable-to-type

It's basically a cast preventing TS to throw an error

Reply:
Interesting... thanks for the info